src/index.ts

```diff
@@ -139,6 +174,7 @@ export class PriceFeed {
     this.expo = rawValues.expo;
     this.id = rawValues.id;
     this.maxNumPublishers = rawValues.maxNumPublishers;
+    this.metadata = rawValues.metadata;
```

Comment:
Why you don't create a PriceFeedMetadata here?

I think the better way would be to have fromJson and toJson in PriceFeedMetadata and call them here. (And remove construction when `getMetadata()` is called)

Reply:
you might want to use `--converters all-objects` in quicktype to generate converter methods for this object as well.